webpack.config.js

```diff
@@ -1,48 +1,52 @@
-var version = require('./package.json').version;
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const path = require('path');
 
-module.exports =
-[
-    {
-        name: "Minified",
-        entry: "./src/embedded.js",
-        output: {
-            path: __dirname + "/dist",
-            filename: "hellosign-embedded." + version + ".min.js",
-            library: "HelloSign",
-            libraryTarget: "var"
-        },
-        module: {
-            loaders: [
-                {
-                    test: /\.js/,
-                    loader: 'uglify',
-                    include: __dirname + "/src"
-                },
-                {
-                    test: /\.json/,
-                    loader: 'json',
-                    include: __dirname
-                }
-            ]
-        }
+const { version } = require('./package.json');
+
+module.exports = [
```

Comment:
I'm wondering if we need to add another output here for NPM users here.  This config will output nicely transpiled browser bundles, but I think if you do a `require('hellosign-embedded')` we don't have any transpiled modules in our distributed NPM package to point to.  I think we could do this via another webpack entry or even just the babel cli with something like `babel ./src/embedded.js -o ./dist/embedded.js`.

Reply:
Yeah good point, I'll add another entry point for npm.

Reply:
Problem with this, however, is that dist files will need to be checked in to source control…

Reply:
So a.) we can either have npm point to the src file, but then we'll have a slight difference between the npm version and the cdn version; or b.) we can create a new output file specifically for npm, but we'll have to check in our dist files with git…

Reply:
Hm, yeah.  I don't think we want to do option A because people using this via NPM will still run into the issue of bundling our raw source into their projects that may not meet their browser support needs.  

I've seen other libraries use a `lib` directory for published code that is transpiled but still in module form.  If we can get this as a pre-publish step before we push to NPM I think we could avoid checking in the compiled code while still getting NPM the right files.  Something like:
- Add `lib` to our `.gitignore` so we're not checking in compiled code
- Add a script in `scripts` that transpiles `src` into `lib` as common JS modules
- Point `main` in `package.json` to `lib/embedded.js`
- In our `travis.yml`, add this new script as a `before_deploy` command

Does that seem right?